docs/DeveloperGuide.md

```diff
@@ -538,6 +542,60 @@ Actor: User
 
     Use case resumes at step 2
 
+### **Use Case: Edit a Reminder for a client**
+
+Actor: User
+
+Preconditions: The client to edit a Reminder for has an existing Reminder.
+
+**MSS**
+1. User requests to list clients.
+2. REP shows a list of clients.
+3. User requests to edit a Reminder for a specific client.
+4. REP updates the Reminder for the client in the system.
+
+**Extensions:**
+* 1a. The list is empty
+
+  Use case ends
+
+* 3a. The given index is invalid
+    * 3a1. REP displays an error message
+
+      Use case resumes at step 2
+
+* 3b. REP detects error in details
+    * 3b1. REP displays an error message
+
+      Use case resumes at step 2
+
+### **Use Case: Delete a Reminder for a client**
+
+Actor: User
+
+Preconditions: The client to edit a Reminder for has an existing Reminder.
+
+**MSS**
+1. User requests to list clients.
+2. REP shows a list of clients.
+3. User requests to delete a Reminder for a specific client.
+4. REP removes the Reminder for the client in the system.
+
+**Extensions:**
+* 1a. The list is empty
+
+  Use case ends
+
+* 3a. The given index is invalid
+    * 3a1. REP displays an error message
+
+      Use case resumes at step 2
+
+* 3b. REP detects error in details
+    * 3b1. REP displays an error message
+
+      Use case resumes at step 2
+
```

Comment:
Should add periods at the end of each point to have consistent formatting.

Reply:
Alright, will do!